Source/Reflection/TypeExtensions.cs

```diff
@@ -68,6 +68,16 @@ public static bool IsNumericType(this Type type)
                    NumericTypes.Contains(Nullable.GetUnderlyingType(type));
         }
 
+        /// <summary>
+        /// Check if a type is enumerable. Note that string is an IEnumerable, but in this case the string is excluded
+        /// </summary>
+        /// <param name="type"><see cref="Type"/> to check</param>
+        /// <returns>True if type is enumerable, false if not an enumerable</returns>
+        public static bool IsEnumerable(this Type type)
```

Comment:
Might be worth just moving this to TypeExtensions.  In any case, the propType.Implements(typeof(IEnumerable)) is all you need.